src/main/java/net/imglib2/algorithm/dog/DifferenceOfGaussian.java

```diff
@@ -97,6 +135,44 @@ public static < I extends NumericType< I >, T extends NumericType< T > & NativeT
 		DoG( sigmaSmaller, sigmaLarger, input, Views.translate( g1, translation ), dog, service );
 	}
 
+	/**
+	 * Compute the difference of Gaussian for the input. Input convolved with
+	 * Gaussian of sigmaSmaller is subtracted from input convolved with Gaussian
+	 * of sigmaLarger (where sigmaLarger &gt; sigmaSmaller).
+	 * </p>
+	 * This method differs from
+	 * {@link #DoG(double[], double[], RandomAccessible, RandomAccessible, RandomAccessibleInterval, ExecutorService)}
+	 * only in that its parameter order is tailored to an Op. The output comes
+	 * last, and the primary input (the input image) comes first.
+	 *
+	 * @implNote op name="filter.DoG", type=Computer
+	 * @param input
+	 *            the input image extended to infinity (or at least covering the
+	 *            same interval as the dog result image, plus borders for
+	 *            convolution).
+	 * @param sigmaSmaller
+	 *            stddev (in every dimension) of smaller Gaussian.
+	 * @param sigmaLarger
+	 *            stddev (in every dimension) of larger Gaussian.
+	 * @param tmp
+	 *            temporary image, must at least cover the same interval as the
+	 *            dog result image.
+	 * @param service
+	 *            how many threads to use for the computation.
+	 * @param dog
+	 *            the Difference-of-Gaussian result image.
+	 */
+	public static < I extends NumericType< I >, T extends NumericType< T > & NativeType< T > > void DoG(
```

Comment:
I wrote these wrappers around the original methods to make the signatures more op-like. What should we do with the older ones? Nothing? Deprecate them?

Reply:
Let's do nothing with the old ones. At most: put a `//` comment near them explaining why they aren't declared as Ops, if you think there is a chance someone will wonder about it later.

Reply:
I already wrote a comment on the ones that *do* have Ops - that's probably enough?